client/.eslintrc.cjs

```diff
@@ -13,9 +13,7 @@ module.exports = {
   plugins: ['react-refresh'],
   rules: {
     'react/jsx-no-target-blank': 'off',
-    'react-refresh/only-export-components': [
-      'warn',
-      { allowConstantExport: true },
-    ],
+    'react-refresh/only-export-components': ['warn', { allowConstantExport: true }],
+    'react/prop-types': 'error',
```

Comment:
not sure why we need this. 
this rules seems to be suitable for TypeScript, but we are dealing with ECMA JavaScript, i doubt we need it.
can you provide any justification, please?

Reply:
To be honest, eslint was throwing errors and asking for props. I installed them and everything worked

Reply:
:) ok
but we should know what we do in every single line of our changes
if you not sure what exactly you are changing - ask me, and we can see why it gets required at your computer but not necessary at my side :)